src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/navbar-logo.html

```diff
@@ -1,19 +1,31 @@
-{% if logo %}
 {% if not theme_logo_link %}
-<a class="navbar-brand" href="{{ pathto(root_doc) }}">
-  <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
-</a>
-{% elif theme_logo_link[:4] == 'http' %}
-<a class="navbar-brand" href="{{ theme_logo_link }}">
-  <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
-</a>
+  {% set href = pathto(root_doc) %}
+{% elif theme_logo_link[:4] == "http" %}
+  {% set href = theme_logo_link %}
 {% else %}
-<a class="navbar-brand" href="{{ pathto(theme_logo_link) }}">
+  {% set href = pathto(theme_logo_link) %}
+{% endif %}
+
+{% if theme_dark_logo and theme_light_logo %}
+
+  <a class="navbar-brand only-light" href="{{ href }}">
+    <img src="{{ pathto('_static/' + theme_light_logo, 1) }}" class="logo" alt="logo">
+  </a>
+
+  <a class="navbar-brand only-dark" href="{{ href }}">
+    <img src="{{ pathto('_static/' + theme_dark_logo, 1) }}" class="logo" alt="logo">
+  </a>
+
+{% elif logo %}
+
+<a class="navbar-brand" href="{{ href }}">
   <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
 </a>
-{% endif %}
+
 {% else %}
-<a class="navbar-brand" href="{{ pathto(root_doc) }}">
-<p class="title">{{ project }}</p>
-</a>
+
+  <a class="navbar-brand" href="{{ href }}">
+    <p class="title">{{ project }}</p>
```

Comment:
how about we ask users to manually define `html_title` if they want to have a title to the right of the logo, so that they can provide both if need be. I realized in https://github.com/pydata/pydata-sphinx-theme/pull/698 that if you have a logo that is relatively small, you might *also* want the title as well. It seems reasonable to tell users that they must use one of `html_logo` or `html_title` and that they can also use both if they wish.